postgres-appliance/scripts/configure_spilo.py

```diff
@@ -718,6 +722,12 @@ def get_placeholders(provider):
     return placeholders
 
 
+def format_url(host, port):
+    if ":" in host:
+        return "[" + host + "]" + ":" + port
```

Comment:
so this does also work with ipv4?

Reply:
Yes, if a host (without port) has `:` in it it indicates IPv6 address and we add the `[]` if not we leave it as is.